auth/checkToken.js

```diff
@@ -0,0 +1,19 @@
+const token = require( 'jsonwebtoken' )
+require( 'dotenv' ).config()
+
+module.exports = ( request, response, next ) => {
+  const authToken = request.body.token || request.query.token || request.headers.authorization || request.headers( 'x-access-token' )
```

Comment:
After reading the code again, I think the second recommendation is preferred - as it stands, the nested if/else makes this a little bit difficult to parse.